scss/bootstrap/components/civicase/_notification-badge.scss

```diff
@@ -0,0 +1,52 @@
+.crm_notification-badge {
+  background: $brand-primary;
+  border-radius: $border-radius-base;
+  color: $crm-white;
+  display: inline-block;
+  font-size: $font-size-base;
+  line-height: 14px;
+  margin: 0 2px;
```

Comment:
I don't think we will have parent Element on each icon. I have used margin to give some space between two icons. Let me know, If you have any other suggestion to achieve this.